lib/vehicle.rb

```diff
@@ -6,17 +6,22 @@ class Vehicle
               :make,
               :model,
               :engine
+              
+  attr_accessor :registration_date,
```

Comment:
You technically added additional methods with this attr_accessor which means there should be additional tests in the spec file to test that we can indeed change these attributes.